docs/tutorials/embed/gcca_tutorial.ipynb

```diff
@@ -11,7 +11,19 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
```

Comment:
seems to be an error

Reply:
yeah this is the error i am getting cause graspy isnt installed